### ISSUE TITLE
To @Mulchman and other developers working on TGD

### ISSUE THREAD

**Issue by dasilva333** [dasilva333/TowerGhostForDestiny#162]
Hey guys I could post this on the wiki but it wouldn't receive as much visibility, I recently made a change to the code that allows me to track all errors and exceptions happening per version. I managed to take care of a lot of the big bugs but there is still some big ones left. This means I won't adding any new features or enhancements until every single problem is cleared. The reason I'm bringing this up to you all is so you can please remember to disable ga.js from your index.html file. 

Let's just say I was a bit confused when I received an error for 
> Uncaught TypeError: tgd.normalizeTemplate is not a function

And yet could not find a single reference to it in my code. :+1: 

TLDR: Remove ga.js from index.html while working on a private branch

PS. If anyone wants to help me tackle some of these crashes happening I can make a separate wiki post with the latest exception logs.

**Comment by Mulchman** [comment]
After commenting out the ga.js line, I get this:

Uncaught ReferenceError: tracking is not definedapp.init @ app.js:1133m.Callbacks.j @ jquery.js:2m.Callbacks.k.fireWith @ jquery.js:2m.extend.ready @ jquery.js:2J @ jquery.js:2

**Comment by dasilva333** [comment]
Good catch sir! Also comment out that line where ever it might be when you work on it. 

> tracking.init();

In the future you won't have to do this because they'll be independent of each other. Thanks!

**Comment by dasilva333** [comment]
Closing this issue, all this stuff is going into: https://github.com/dasilva333/TowerGhostForDestiny/blob/master/CONTRIBUTING.md
